pkg/process/processors/github_processor.go

```diff
@@ -51,7 +51,7 @@ func (p githubProcessor) IsSupported(schemaURL string) bool {
 		return false
 	}
 
-	if !strings.HasSuffix(schemaURL, "schema-1.0.0.json") {
+	if !strings.HasSuffix(schemaURL, "schema-1.0.0.json") && !strings.HasSuffix(schemaURL, "schema-1.0.1.json") {
```

Comment:
soon we can replace this with a version extraction and comparison -- we can keep doing this for this PR though.